csrc/codebase/mmseg/segment.cpp

```diff
@@ -25,6 +25,10 @@ class ResizeMask : public MMSegmentation {
     MMDEPLOY_DEBUG("preprocess: {}\ninference: {}", preprocess_result, inference_result);
 
     auto mask = inference_result["output"].get<Tensor>();
+    auto mask_src = cpu::Tensor2CVMat(mask);
```

Comment:
`mask` may be a tensor on GPU. And it may be of type INT64 for onnxruntime, which is not a supported data type in OpenCV.